30-inheritance.cs

```diff
@@ -66,7 +65,6 @@ public string Calculate()
             return "P";
         if (55 > average && average >= 40)
```

Comment:
always use a variable in the first place aka 
`if (average < 55 && average >= 40)`